### PR TITLE
[docs] update macos installation guide 2

### PR DESCRIPTION
**Pull request by Jacob27**
### Details:
 - *update macos installation guide*

### Tickets:
 - *Cvs 71844*

### CODE REVIEW

File: docs/install_guides/installing-openvino-macos.md [hunk 1]
```diff
@@ -1,212 +1,156 @@
...
+@endsphinxdirective
 
 ## Overview
 
 This guide provides step-by-step instructions on how to install the Intel® Distribution of OpenVINO™ toolkit for macOS*. The following steps will be covered:
 
 1. <a href="#install-core">Install the Intel® Distribution of OpenVINO™ Toolkit</a>
 2. <a href="#set-the-environment-variables">Configure the Environment</a>
-3. <a href="#model-optimizer">Configure the Model Optimizer</a>
+3. <a href="#model-optimizer">Download additional components (Optional)</a>
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-macos.md:41]: Can we add here information on how to download OpenCV using a script supplied by macOS installer?

@mshabunin to provide instructions.

File: docs/install_guides/installing-openvino-macos.md [hunk 2]
```diff
@@ -112,9 +112,9 @@ Developing in Python:
    * [Start with PaddlePaddle models with OpenVINO](https://docs.openvino.ai/latest/notebooks/103-paddle-onnx-to-openvino-classification-with-output.html)
 
 Developing in C++:
-   * [Image Classification Async C++ Sample](../../samples/cpp/classification_sample_async/README.md)
-   * [Hello Classification C++ Sample](../../samples/cpp/hello_classification/README.md)
-   * [Hello Reshape SSD C++ Sample](../../samples/cpp/hello_reshape_ssd/README.md)    
+   * [Image Classification Async C++ Sample](https://docs.openvino.ai/latest/openvino_inference_engine_samples_classification_sample_async_README.html)
+   * [Hello Classification C++ Sample](https://docs.openvino.ai/latest/openvino_inference_engine_samples_hello_classification_README.html)
+   * [Hello Reshape SSD C++ Sample](https://docs.openvino.ai/latest/openvino_inference_engine_samples_hello_reshape_ssd_README.html)
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-macos.md:117]: Our documentation is versioned. I believe we should use global links format here.  It's permanently points to the `latest`, whereas relative path routes to corresponding version of a page.
CC'ed: @andrew-zaytsev

Comment (Jacob27) [reply docs/install_guides/installing-openvino-macos.md:117]: @slyubimt I agree but when I tried to use relative paths it didn't work on built documentation, you can check my attempt at previous commit https://github.com/openvinotoolkit/openvino/pull/9636/commits/76046a239541cc172869aa32b94c690dfb8e5adf

Comment (slyubimt) [reply docs/install_guides/installing-openvino-macos.md:117]: @ntyukaev Can you help with this issue?

File: docs/install_guides/installing-openvino-macos.md [hunk 3]
```diff
@@ -1,212 +1,156 @@
...
 
 ## <a name="install-core"></a>Step 1: Install the Intel® Distribution of OpenVINO™ Toolkit Core Components
 
-If you have a previous version of the Intel® Distribution of OpenVINO™ toolkit installed, rename or delete these two directories:
-
-- `~/inference_engine_samples`
-- `~/openvino_models`
-
-[Download the latest version of OpenVINO toolkit for macOS*](https://software.intel.com/en-us/openvino-toolkit/choose-download/free-download-macos), then return to this guide to proceed with the installation.
+1. Download the Intel® Distribution of OpenVINO™ toolkit package file from [Intel® Distribution of OpenVINO™ toolkit for macOS*](https://software.intel.com/en-us/openvino-toolkit/choose-download/free-download-macos). Select the Intel® Distribution of OpenVINO™ toolkit for macOS package from the dropdown menu.
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-macos.md:47]: why didn't we add animated process here as it was done for Windows & Linux?

File: docs/install_guides/installing-openvino-macos.md [hunk 4]
```diff
@@ -1,212 +1,156 @@
...
-7. **Optional**: You can choose **Customize** to change the installation directory or the components you want to install.
-> **NOTE**: If there is an OpenVINO™ toolkit version previously installed on your system, the installer will use the same destination directory for next installations. If you want to install a newer version to a different directory, you need to uninstall the previously installed versions.
-
-8. The **Finish** screen indicates that the core components have been installed:
-   ![](../img/openvino-install-macos-05.png)
-
-   Once you click **Finish** to close the installation wizard, a new browser window opens with the document you’re reading now (in case you installed without it) and jumps to the section with the next installation steps.
+   For simplicity, a symbolic link to the latest installation is also created: `<INSTALL_DIR>/intel/openvino_2022/`.
 
 The core components are now installed. If you received a message that you were missing external software dependencies (list available under Software Requirements at the top of this guide), you need to install them now before continuing to the next section.
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-macos.md:64]: This is leftover message for previous installer version. Please remove it or rework.

**Review by slyubimt — CHANGES_REQUESTED** [review]

**Comment by Jacob27** [comment]
http://openvino-docs.inn.intel.com/jacek-linux-installation-guide-3/openvino_docs_install_guides_installing_openvino_macos.html

File: docs/install_guides/installing-openvino-macos.md [hunk 5]
```diff
@@ -1,212 +1,153 @@
...
+   * [Start with PaddlePaddle models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/103-paddle-onnx-to-openvino-classification-with-output.html)
 
-For example, to install the `libusb` library using Homebrew\*, use the following command:
-```sh
-brew install libusb
-```
+Developing in C++:
+   * [Image Classification Async C++ Sample](@ref openvino_inference_engine_samples_classification_sample_async_README)
+   * [Hello Classification C++ Sample](@ref openvino_inference_engine_samples_hello_classification_README)
+   * [Hello Reshape SSD C++ Sample](@ref openvino_inference_engine_samples_hello_reshape_ssd_README)
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-macos.md:114]: Now you are ready to try out the toolkit. You can use the following tutorials to write your applications using Python and C++.

Python:

C++:

File: docs/install_guides/ncs2-setup-guide.md [hunk 6]
```diff
@@ -6,6 +6,8 @@
 
 @endsphinxdirective
 
+## Linux
+
 Once you have your Intel® Distribution of OpenVINO™ toolkit installed, follow the steps to be able to work on NCS2:
 
 1. Go to the install_dependencies directory:
```
Comment (slyubimt) [inline docs/install_guides/ncs2-setup-guide.md:13]: <INSTALL_DIR>/install_dependencies/

**Review by slyubimt — CHANGES_REQUESTED** [review]

File: docs/install_guides/ncs2-setup-guide.md [hunk 7]
```diff
@@ -20,3 +22,25 @@ Once you have your Intel® Distribution of OpenVINO™ toolkit installed, follow
...
+
+.. _ncs guide macos:
+
+@endsphinxdirective
+
+## macOS
+
+These steps are required only if you want to perform inference on Intel® Neural Compute Stick 2
+powered by the Intel® Movidius™ Myriad™ X VPU. For more details, see also the
+[Get Started page for Intel® Neural Compute Stick 2](https://software.intel.com/en-us/neural-compute-stick/get-started).
```
Comment (slyubimt) [inline docs/install_guides/ncs2-setup-guide.md:36]: Can we rid of this outdated page?

File: docs/install_guides/installing-openvino-macos.md [hunk 8]
```diff
@@ -1,212 +1,153 @@
...
-   ```
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. The OpenVINO™ Model Development Tools can only be installed via PyPI now. See [Install OpenVINO™ Model Development Tools](@ref installing_model_dev_tools) for detailed steps.
+Among the model developer tools you can find e.g. Model Optimizer which imports, converts, and optimizes models, and Model Downloader which gives you access to a collection of pre-trained deep learning public and intel-trained models.  
 
-2. Run the script to configure the Model Optimizer for Caffe, TensorFlow, MXNet, Kaldi\*, and ONNX:
-   ```sh
-   sudo ./install_prerequisites.sh
-   ```
+To enable inference on Intel® Neural Compute Stick 2, see the <a href="#configure-ncs2">Configure the Intel® Neural Compute Stick 2 </a>. Or proceed to the <a href="#get-started">What’s next?</a> section to learn the basic OpenVINO™ toolkit workflow and run code samples and demo applications.
```
[the remark this answers is not in the document]
Comment (Jacob27) [reply docs/install_guides/installing-openvino-macos.md:91]: yes, this indicates what to do next when user is done with Step 2. The user can move to Step 3 concerning NCS2 or go to What's next section.

File: docs/install_guides/installing-openvino-macos.md [hunk 9]
```diff
@@ -1,212 +1,153 @@
...
-
-The Model Optimizer is configured for one or more frameworks.
+## <a name="configure-ncs2"></a>Step 4 (Optional): Configure the Intel® Neural Compute Stick 2 
 
-You have completed all required installation, configuration and build steps in this guide to use your CPU to work with your trained models. 
+@sphinxdirective
 
-To enable inference on Intel® Neural Compute Stick 2, see the <a href="#configure-ncs2">Configure the Intel® Neural Compute Stick 2 </a>. 
+Only if you want to perform inference on Intel® Neural Compute Stick 2 powered by the Intel® Movidius™ Myriad™ X VPU, follow the steps on :ref:`NCS2 Setup Guide <ncs guide macos>`.
+For more details, see the `Get Started page for Intel® Neural Compute Stick 2 <https://software.intel.com/en-us/neural-compute-stick/get-started>`_.
```
[the remark this answers is not in the document]
Comment (Jacob27) [reply docs/install_guides/installing-openvino-macos.md:98]: I'm still waiting for feedback concerning this part from @mlyashko

File: docs/install_guides/ncs2-setup-guide.md (hunk 7 above)
Comment (Jacob27) [reply docs/install_guides/ncs2-setup-guide.md:36]: let me synch with @mlyashko regarding this

File: docs/install_guides/installing-openvino-macos.md (hunk 1 above)
Comment (slyubimt) [reply docs/install_guides/installing-openvino-macos.md:41]: macOS3

File: docs/install_guides/installing-openvino-macos.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (slyubimt) [reply docs/install_guides/installing-openvino-macos.md:91]: The point is that two paragraphs of this section are logically unrelated and do not match the heading: _(Optional): Download additional components._

File: docs/install_guides/installing-openvino-macos.md [hunk 10]
```diff
@@ -1,212 +1,153 @@
...
 
-   **Optional:** If you don't want to change your shell profile, you can run the following script to temporarily set your environment variables when working with the OpenVINO* toolkit:
+If you have more than one OpenVINO™ version on your machine, you can easily switch its version by sourcing `setupvars.sh` of your choice.
 
-   ```sh
-   source /opt/intel/openvino_2021/bin/setupvars.sh
-   ```  
+> **NOTE**: You can also run this script every time when you start new terminal session. Open `~/.bashrc` in your favorite editor, and add `source <INSTALL_DIR>/intel/openvino_2022/bin/setupvars.sh`. Next time when you open a terminal, you will see `[setupvars.sh] OpenVINO™ environment initialized`. Changing `.bashrc` is not recommended when you have many OpenVINO™ versions on your machine and want to switch among them, as each may require different setup.
 
 The environment variables are set. Continue to the next section to configure the Model Optimizer.
```
Comment (slyubimt) [inline docs/install_guides/installing-openvino-macos.md:84]: `Continue to the next section to configure the Model Optimizer.` - this step is not relevant any more.

File: docs/install_guides/installing-openvino-macos.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (Jacob27) [reply docs/install_guides/installing-openvino-macos.md:91]: thank you @slyubimt, I've consulted also validation team and I'm going to remove this part not to confuse anyone in future

**Comment by Jacob27** [comment]
http://openvino-docs.inn.intel.com/jacek-linux-installation-guide-3/openvino_docs_install_guides_installing_openvino_macos.html

**Review by slyubimt — CHANGES_REQUESTED** [review]

**Comment by Jacob27** [comment]
http://openvino-docs.inn.intel.com/jacek-linux-installation-guide-3/openvino_docs_install_guides_installing_openvino_macos.html

**Review by slyubimt — APPROVED** [review]

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

**Review by andrew-zaytsev — APPROVED** [review]

File: docs/install_guides/installing-openvino-windows.md [hunk 11]
```diff
@@ -84,7 +84,7 @@ The core components are now installed. Continue to the next section to configure
 You must update several environment variables before you can compile and run OpenVINO™ applications. Open the Command Prompt, and run the `setupvars.bat` batch file to temporarily set your environment variables:
 
 ```sh
-"<INSTALL_DIR>\openvino_2022\setupvars.bat"
+"<INSTALL_DIR>\setupvars.bat"
```
[the remark this answers is not in the document]
Comment (Jacob27) [reply docs/install_guides/installing-openvino-windows.md:87]: @slyubimt could you confirm this comment is valid for windows?

[the remark this answers is not in the document]
Comment (slyubimt) [reply docs/install_guides/installing-openvino-windows.md:87]: Yes, It would be nice to point the default installation path on Windows:
C:\Program Files (x86)\Intel\openvino_<version>
and symbolic link:
C:\Program Files (x86)\Intel\openvino_2022
